lib/Models/WhyAmISpecialFunction.js

```diff
@@ -29,6 +29,8 @@ var WhyAmISpecialFunction = function(terria) {
         name: 'Region Type',
         description: 'The type of region to analyze.'
     });
+    // Let the RegionParameter handle showing it in the UI, rather than displaying it directly.
+    this._regionTypeParameter.showInUi = false;
```

Comment:
We can probably do this automatically at the UI level.  Look for a `RegionParameter` with a `regionType` property equal to this `RegionTypeParameter`.  If we find one, don't show the UI for this `RegionTypeParameter`.